.github/workflows/ci.yaml

```
@@ -128,9 +133,9 @@ jobs:
         if: ${{ !matrix.devel }}
         run: git diff --exit-code
 
-  check:  # This job does nothing and is only used for the branch protection
+  check: # This job does nothing and is only used for the branch protection
```

Keep two space before a comment requirement, just like in Python.
```suggestion
  check:  # This job does nothing and is only used for the branch protection
```
Plus note that this chunk of the workflow is synchronized across multiple repositories and changing it unnecessarily increases the maintenance burden because this causes merge conflicts when the updates to this chunk are distributed into the repositories that have a copy of it.